index.html

```diff
@@ -4,9 +4,78 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="./styles/index.css" rel="stylesheet"/>
+    <script src="./node_modules/axios/dist/axios.min.js"></script>
+    <script src="./src/index.js"></script>
     <title>Weather Report</title>
 </head>
 <body>
-    
+    <div id="headerSection">
+        <header>Weather report </header>
```

Comment:
Since this is the title of the page it should be a `h1`:

```suggestion
        <header><h1>Weather report</h1></header>
```